artifact_requirements.txt

```diff
@@ -0,0 +1,18 @@
+# Note: This requires users to update the branch to create an envrionment from the branch they are
+#   working on.
+vivarium_gates_nutrition_optimization_child @ git+https://github.com/ihmeuw/vivarium_gates_nutrition_optimization_child@main
+vivarium @ git+https://github.com/ihmeuw/vivarium@release-candidate-spring
+vivarium_public_health @ git+https://github.com/ihmeuw/vivarium_public_health@release-candidate-spring
+gbd_mapping>=4.0.0
+click
+jinja2
+loguru
+numpy
+pandas
+pyyaml
+scipy
+tables
+vivarium_inputs[data]>=5.0.0
+vivarium_cluster_tools @ git+https://github.com/ihmeuw/vivarium_cluster_tools@release-candidate-spring
+black==22.3.0
```

Comment:
This seems like a bunch of things aren't necessary for artifact-building (or at the least, black and isort).

Reply:
The artifacts.txt is linked in the artifact env shell script where the simulation environment creation shell script links to the normal requirements.txt